test/browser/support/env.js

```diff
@@ -2,16 +2,17 @@ const { setWorldConstructor } = require("@cucumber/cucumber");
 
 require("playwright");
 
-const users = {
-  "A user": {},
+const userClaims = {
+  "a user": require("../support/shared_claim"),
```

Comment:
Do you think we should update this shared claims to be Yasmine Young rather than a name that is not going to work at all?